build-logic/src/main/kotlin/org/jetbrains/conventions/gradle-plugin.gradle.kts

```diff
@@ -0,0 +1,8 @@
+package org.jetbrains.conventions
+
+plugins {
+    id("org.gradle.kotlin.kotlin-dsl")
+    id("org.jetbrains.conventions.maven-publish")
+    id("org.jetbrains.conventions.kotlin-jvm")
+    id("com.gradle.plugin-publish")
+}
```

Comment:
Was there a need to introduce this convention, did it interfere with the update somehow? 

If not, I have some concerns about why it is needed and whether it breaks something now or in the future (`kotlin-jvm` was previously not part of the gradle runner)

Reply:
`kotlin-dsl` added `plugin.sam.with.receiver.gradle.plugin` and other plugins with default version 1.7.10 instead of 1.9.0, KGP should align their version to 1.9.0, but it did not work.